src/styles/gameBoard.scss

```diff
@@ -0,0 +1,23 @@
+h3 {
+  margin: 15px 0;
+}
+
+h4 {
+  margin: 15px 0;
+}
```

Comment:
This is VERY dangerous. You are eventually changing global h3 and h4 style.
You should only have a top level style guide to define how each html tag should behave then always add a class to any html tag afterwards.